### PR TITLE
Update usage.rst

### PR DESCRIPTION
**Pull request by bradfordcondon**
change menu to match menu path

## Metadata

 - Issue #

- [ ] I feel this PR is ready to be merged.
- [ ] This PR is dependent upon [PR #/ nothing]

Documentation:
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

## Testing?

- [ ] I tested on a generic Tripal Site
- [ ] I tested on a KnowPulse Clone
- [ ] This PR includes automated testing

### CODE REVIEW

**Review by laceysanderson — COMMENTED** [review]
It's actually supposed to be `Set-up Ontologies` :-)

**Comment by bradfordcondon** [comment]
https://github.com/UofS-Pulse-Binfo/analyzedphenotypes/blob/e5241aadc9f7366dea39d3f584dfa03eb7359b6e/analyzedphenotypes.module#L89

yeah... possible i needed to clear cache after updating or something similar.

File: docs/admin_guide/usage.rst [hunk 1]
```diff
@@ -3,7 +3,7 @@ General Usage
 
 .. warning::
 
-  Before you can use this module, you need to set it up the module by going to Admin > Tripal > Extensions> Analyzed Phenotypes > Set-up. For more detail on how to do this, see the "Setup" documentation in the Administrators Guide.
+  Before you can use this module, you need to set it up the module by going to Admin > Tripal > Extensions> Analyzed Phenotypes > Configuration. For more detail on how to do this, see the "Setup" documentation in the Administrators Guide.
```
Comment (laceysanderson) [inline docs/admin_guide/usage.rst]:
```suggestion
  Before you can use this module, you need to set it up the module by going to Admin > Tripal > Extensions> Analyzed Phenotypes > Set-up Ontologies. For more detail on how to do this, see the "Setup" documentation in the Administrators Guide.
```

**Comment by laceysanderson** [comment]
I confirmed it should be `Set-up Ontologies` :-)

**Pull request closed without merging**